spammers.txt

```diff
@@ -68,6 +75,7 @@ animenime.ru
 anticrawler.org
 antiguabarbuda.ru
 apteka-pharm.ru
+archive.fo
```

Comment:
This seems like a legitimate domain? Certainly not a spammer.

Reply:
Yeah there are a few legitimate domains in this list still imo. I am going to close this PR since it's a bit dated and has conflicted.